plugins/plugins/commands

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
+case "$1" in
+  plugins-install)
+    pluginhook install
+    exit $?
+    ;;
+
+  plugins-install-dependencies)
+    pluginhook dependencies
+    exit $?
+    ;;
+
+  plugins-update)
+    pluginhook update
+    exit $?
+    ;;
+
+  help)
+    :
+    ;;
+esac
+
+if [ -z $DOKKU_NO_COMMAND ]; then
+  case "$1" in
+    nginx:*)
+      PLUGIN_NAME="nginx-vhosts"
+      ;;
+    *)
+      PLUGIN_NAME="$1"
+      ;;
+  esac
+  if [ -x $PLUGIN_PATH/$PLUGIN_NAME/command ]; then
+    "$PLUGIN_PATH/$PLUGIN_NAME/command" "$@"
```

Comment:
Yes and yes.

We still for loop here: https://github.com/progrium/dokku/blob/592_mh-plugin-structure-refactor/dokku#L157-L167

The `core`, `git`, and `plugins` plugins still have `commands` files.